clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -272,20 +272,11 @@ export class HttpConnection implements IConnection {
             throw new Error(`Cannot resolve '${url}'.`);
         }
 
-        const parser = window.document.createElement("a");
-        parser.href = url;
+        const aTag = window.document.createElement("a");
+        aTag.href = url;
```

Comment:
Because this can't easily be unit tested I think there should be a comment describing the different URL scenarios that this logic normalizes to keep future devs in the loop